doc/en/reference.rst

```diff
@@ -1010,6 +1010,23 @@ loaded.
 This is not meant to be set by users, but is set by pytest internally with the name of the current test so other
 processes can inspect it, see :ref:`pytest current test env` for more information.
 
+.. envvar:: PY_COLORS
+
+When set to "1", pytest will use color in terminal output.
+When set to "0", pytest will not use color.
```

Comment:
Or?

```suggestion
When set to ``1``, pytest will use color in terminal output.
When set to ``0``, pytest will not use color.
```

# After

![image](https://user-images.githubusercontent.com/1324225/87016990-16553100-c1d8-11ea-8f84-83968f84fffc.png)

Reply:
I like this one better. 👍